DockDoor/Utilities/KeybindHelper.swift

```diff
@@ -117,20 +117,18 @@ class KeybindHelper {
     }
     
     private func showHoverWindow() {
-        Task { [weak self] in
-            do {
-                guard let self = self else { return }
-                let windows = try await WindowUtil.activeWindows(for: "")
-                await MainActor.run { [weak self] in
-                    guard let self = self else { return }
-                    if self.isModifierKeyPressed {
-                        SharedPreviewWindowCoordinator.shared.showWindow(appName: "Alt-Tab", windows: windows, overrideDelay: true,
-                                                                         centeredHoverWindowState: .windowSwitcher, onWindowTap: { SharedPreviewWindowCoordinator.shared.hideWindow() })
-                    }
+        let apps = NSWorkspace.shared.runningApplications
+        let windows = apps.flatMap { app in
+            return WindowUtil.getRunningAppWindows(app: app)
```

Comment:
Ensure you use `[weak self]` in the closure like the original code, otherwise there may be memory retention issues. The app previously had a memory leak/abandoned memory problem, which was resolved by using weak self references. The new code removes this safety measure, which could reintroduce memory management issues.

If you do decide to keep this change, I would just be sure to run lengthy tests to ensure that memory usage doesn't grow continuously as the app keeps running and being used.